src/styles/blocks/page.scss

```diff
@@ -0,0 +1,79 @@
+.page {
+  font-family: "Roboto", sans-serif;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 27px;
+  color: #3e3e3e;
+
+  scroll-behavior: smooth;
+
+  &__body {
+    margin: 0;
+    background: $c-gray;
```

Comment:
That's not right background color, check [figma](https://www.figma.com/file/nHz8bflIwJaWP3P99vKTH5/miami_home_new?node-id=16033%3A3)

Reply:
ready